src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -36,21 +43,42 @@ const TopicDomainHeader = (props) => {
             marginRight: normalize(5),
             height: normalize(8)
           }}
+          width={isPreview ? 9 : 16}
+          height={isPreview ? 9 : 16}
         />
         {memberCount === undefined && props.isLoading ? (
           <Shimmer height={10} width={normalize(25)} />
         ) : (
-          <Text style={styles.domainMember(shouldDisplay)}>{memberCount}</Text>
+          <Text
+            style={[
+              styles.domainMember(shouldDisplay),
+              isPreview ? {fontSize: normalizeFontSize(9)} : {}
+            ]}>
+            {memberCount}
+          </Text>
         )}
-        <Text style={styles.domainMember(shouldDisplay)}> Members</Text>
+        <Text
+          style={[
+            styles.domainMember(shouldDisplay),
+            isPreview ? {fontSize: normalizeFontSize(9)} : {}
+          ]}>
+          {' '}
+          Members
+        </Text>
       </View>
       {props?.initialData?.memberCount === undefined && props.isLoading ? (
         <Shimmer height={10} width={normalize(60)} />
       ) : (
         isFollow &&
         !hideSeeMember && (
           <Pressable onPress={handlePress} style={{backgroundColor: COLORS.transparent}}>
-            <Text style={styles.seeMemberText(followType)} numberOfLines={1} ellipsizeMode="tail">
+            <Text
+              style={[
+                styles.seeMemberText(followType),
+                isPreview ? {fontSize: normalizeFontSize(9)} : {}
+              ]}
+              numberOfLines={1}
+              ellipsizeMode="tail">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The same suggestion applies here as well. Move the conditional styling logic based on `isPreview` prop into the respective style functions. This will make the JSX cleaner and more maintainable.



<!-- This is an auto-generated comment by OSS CodeRabbit -->